man/org.freedesktop.systemd1.xml

```diff
@@ -2782,6 +2782,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly t MemorySwapPeak = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly t MemoryZswapCurrent = ...;
```

Comment:
we so far spelled this as "ZSwap" rather than "Zswap". Compare `MemoryZSwapMax`

Reply:
-> #29999.

Reply:
Darn. I actually looked up how upstream writes it and went with that, while forgetting to look for prior-art in systemd.